Solutions/Infoblox Cloud Data Connector/Analytic Rules/HighNumberofNXDOMAINDNSQueriesDetected.yaml

```diff
@@ -0,0 +1,39 @@
+id: b49637ac-36cc-451c-a67d-bb08a66c0a22
+name: High Number of NXDOMAIN DNS Queries Detected
+description: |
+  'This creates an incident in the event a host generates a high number of DNS queries for non-existent domains.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: InfobloxCloudDataConnector
+    dataTypes: 
+      - CommonSecurityLog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1498
+  - T1565
+query: |
+
+  let timeframe = 1h;
+  let threshold = 200;
+  CommonSecurityLog
+  | where DeviceVendor == "Infoblox" and DeviceProduct == "Data Connector"
+  | where DeviceEventClassID contains "RPZ"
```

Comment:
If possible, use ```has``` instead of ```contains```.  If it is the start or end of a string you can also use ```hasprefix``` or ```hassuffix```.